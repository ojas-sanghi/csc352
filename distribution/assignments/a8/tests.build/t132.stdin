
1 
6 2 
3 7 
7 
1 13 
22 22 
4 
7 11 
15 

19 
10 
28 19 
21 0 
17 6 


7 
29 
17 
20 24 

19 
8 27 


29 5 

11 
22 2 
16 19 
