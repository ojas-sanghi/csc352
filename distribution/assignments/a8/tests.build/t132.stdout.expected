((-0-)1(((-1-)2((-2-)3(-4(-5-))))6((-6-)7(-7((-7(((-7(-8-))10-)11(-11-)))13((-15(((-16-)17(-17-))19(-19(((-19(-19-))20-)21-))))22(-22(((-22-)24(-27-))28(-29(-29-))))))))))
Insert 10
((-0-)1(((-1-)2((-2-)3(-4(-5-))))6((-6-)7(-7((-7(((-7(-8-))10(-10-))11(-11-)))13((-15(((-16-)17(-17-))19(-19(((-19(-19-))20-)21-))))22(-22(((-22-)24(-27-))28(-29(-29-))))))))))
